templates/payments/event-cancelled-refund-request.html

```diff
@@ -0,0 +1,60 @@
+{% from "_formhelpers.html" import render_field %}
+{% extends "base.html" %}
+{% block title %}Refund Request{% endblock %}
+{% block body %}
+
+<div class="col-md-10">
+
+<h2>Refund your payment</h2>
+
+<p>You can request a refund for your payment with this form. We'll let you know by email when it's approved.</p>
+
+<p>If you have any questions,
+   please <a href="mailto:{{ config['TICKETS_EMAIL'][1] }}?subject=Refund Query (WEB-{{payment.id}})">drop us an email</a>.</p>
+
+<div class="well">
+<form method="post" action="{{ url_for('.payment_refund_request', payment_id=payment.id, currency=currency) }}">
+{{ form.hidden_tag() }}
+  <fieldset>
+    <legend>Donation</legend>
+    <p>If you want, you can donate some of your payment to help us recoup our losses for this year
+        and fund our next event. This is completely optional &mdash; we have enough money in the bank
+        to survive until 2022.</p>
```

Comment:
```suggestion
        to survive until 2026.</p>
```

Reply:
(maybe, unless this template is now entirely for historical purposes; in which case perhaps we should rely on `git` history for that instead, and drop it?)